libs/state-machine/src/lib/getEmptySchemaData.ts

```diff
@@ -23,6 +23,8 @@ export const getEmptySchemaData = (
 			mod[key] = 0;
 		} else if (zod instanceof ZodBoolean) {
 			mod[key] = false;
+		} else if (zod instanceof ZodDate) {
+			mod[key] = new Date();
```

Comment:
Is it possible to have a date field without defaulting it to today?  In my past experience, if a user is unsure what value something should have they often just accept the default, no matter how inappropriate.  If it's mandatory to have a default, then it might be better to be e.g. two week's time or a month's time, rather than today

Reply:
👍 Date fields can default to 'undefined' - this does mean that when the field is not optional on the schema, the form starts out with a validation error, but that's probably better than using an inappropriate default.

I think with a bit of work, we could define defaults on the schema for particular fields, but that's one for another PR!